packages/client/src/views/OIDPVerificationCallback/utils.ts

```diff
@@ -63,8 +66,8 @@ interface UserInfo {
 
 interface INidCallbackState {
   pathname: string | undefined
-  declarationId: string | undefined
-  section: string | undefined
+  declarationId: string
+  section: string
```

Comment:
Decided to assert that these are always in the query parameters, and let the software throw an error if they're not, as it should never happen.